aicsimageio/aics_image.py

```diff
@@ -58,14 +61,18 @@ class AICSImage:
 
     Initialize an image and pass arguments to the reader using kwargs.
 
-    >>> img = AICSImage("my_file.czi", chunk_by_dims=["T", "Y", "X"])
+    >>> img = AICSImage("my_file.czi", chunk_dims=["T", "Y", "X"])
 
     Initialize an image, change scene, read data to numpy.
 
     >>> img = AICSImage("my_many_scene.czi")
     ... img.set_scene("Image:3")
     ... img.data
 
+    Initialize an image with a specific reader.
+
+    >>> img = AICSImage("malformed_metadata.ome.tiff", reader=readers.TiffReader)
```

Comment:
This needs a short comment pointing out that you are purposely using the wrong reader because you know in advance the metadata is malformed and OmeTiffReader will raise.

Reply:
might even be clearer if you have .tiff but use OmeTiffReader because you know it is really OME?  Or a s3 guid file name that has no file extension :)

Reply:
The problem I have will both of your examples above is that we handle them both by default already.


Example 1: OmeTiffReader -> TiffReader rollback
this is already done as a part of normal operations, this is just speeding up the file checking / validation by skipping OmeTiffReader basically.

Example 2: a tiff missing the ome in the extension
this is already done because OmeTiffReader is checked before TiffReader so we naturally pick up these files. In fact we have tests for that exact case.

I will add a comment that basically explains example 1 to the docstring.

"Initialize an image with a specific reader. In this case, the assumption is that even though the extension indicates the file is an OME-TIFF file, AICSImage fails to validate the OME metadata. We naturally handle this and will roll back to TiffReader in cases like this _but_ if you don't want to see and log errors / warnings then we can skip that always erroring validation and extra work and skip straight to TiffReader usage."

Reply:
All it really has to say is "This is useful if you know the file type in advance, or would like to skip a few of the file format checks we do internally, which could cause network round trips for remote files."